EveHQ.Prism/Classes/StructureBonus.vb

```diff
@@ -0,0 +1,229 @@
+﻿Imports EveHQ.EveData
+
+''' <summary>
+''' A Structure Bonus Definition
+''' </summary>
+Public Class StructureBonus
+
+    Private _group As String
+    Private _materialEfficiency As Decimal
+    Private _timeEfficiency As Decimal
+    Private _costEfficiency As Decimal
+    Private _scienceBonus As Boolean
+
+    ''' <summary>
+    ''' The Group to which the Bonus should be Applied
+    ''' </summary>
+    Public Property Group As String
+        Get
+            Return _group
+        End Get
+        Set(value As String)
+            _group = value
+        End Set
+    End Property
+
+    ''' <summary>
+    ''' The Material Efficiency Rate provided Bonus
+    ''' </summary>
+    Public Property MaterialEfficiency As Decimal
+        Get
+            Return _materialEfficiency
+        End Get
+        Set(value As Decimal)
+            _materialEfficiency = value
+        End Set
+    End Property
+
+    ''' <summary>
+    ''' The Time Efficiency Rate provided Bonus
+    ''' </summary>
+    Public Property TimeEfficiency As Decimal
```

Comment:
I can drop the boolean IsScientific and create two fully distinct property in order to store bonuses ?

I don't really figure what will be in `StructureScienceBonus` and in `ManufacturingBonus` since all methods/properties are shared (except the CE atm, but it could be added by a rigs for manufacturing jobs in a future, so...)